data/language/english_uk.txt

```diff
@@ -4039,8 +4038,20 @@ STR_5732    :Anisotropic
 STR_5733    :Use NN scaling at integer scales
 # tooltip for tab in options window
 STR_5734    :{SMALLFONT}{BLACK}Rendering
-STR_5735    :Network Status
-STR_5736    :Player
```

Comment:
There has got to be a better way to implement these strings.  This file has been the single largest headache that I've had trying to get this merged. Now I have to go redo TWO files AGAIN because a change happened after the first rebase I did.  I have to go and move all my string numbers down because a different change was merged before mine.  Does this happen for almost literally every merged PR that touches this file?  

I mean, I'll fix it, but it seems like this is way more of a headache than it needs to be.